core/server/services/auth/setup.js

```diff
@@ -110,14 +110,15 @@ async function doProduct(data, productsAPI) {
         return user;
     }
     try {
-        const page = await productsAPI.browse({limit: 1});
+        const page = await productsAPI.browse({limit: 'all'});
```

Comment:
Need to fetch all, because we cannot assume first one is free - generally there will be 2